tests/Feature/Posts/DeletePostsTest.php

```diff
@@ -12,42 +12,48 @@
 
 class DeletePostsTest extends TestCase
 {
-    use WithFaker;
-    use DatabaseTransactions;
-
-    private $utility;
-
-    protected function setUp(): void
-    {
-        parent::setUp();
-        $this->utility = new Utility($this);
-        $this->utility->testSetup();
-    }
-
     /** @test */
     public function it_does_not_allow_non_auth_users_to_delete_post()
     {
-        $post = Post::first();
+        $post = factory(Post::class)->create();
         $this->deleteJson(route('api.posts.delete', $post->id))
             ->assertUnauthorized();
     }
 
     /** @test */
-    public function it_does_not_allow_guest_to_delete_post()
+    public function it_does_not_allow_auth_user_to_delete_post()
```

Comment:
This name is misleading, maybe change it to `it_does_not_allow_a_user_to_delete_another_users_post`